docs/kubeadm.md

```diff
@@ -0,0 +1,212 @@
+
+* If you want to be able to schedule workloads on the master, for example if you want a single-machine Kubernetes cluster for development, run:
+
+      # kubeadm init --schedule-workload
+
+* If you do not want to be able to schedule workloads on the master (perhaps for security reasons), run:
+
+      # kubeadm init
+
+This will download and install the cluster database and "control plane" components.
+This may take several minutes.
+
+The output should look like:
+
+    <master/tokens> generated token: "f0c861.753c505740ecde4c"
+    <master/pki> created keys and certificates in "/etc/kubernetes/pki"
+    <util/kubeconfig> created "/etc/kubernetes/kubelet.conf"
+    <util/kubeconfig> created "/etc/kubernetes/admin.conf"
+    <master/apiclient> created API client configuration
+    <master/apiclient> created API client, waiting for the control plane to become ready
+    <master/apiclient> all control plane components are healthy after 61.346626 seconds
+    <master/apiclient> waiting for at least one node to register and become ready
+    <master/apiclient> first node is ready after 4.506807 seconds
+    <master/discovery> created essential addon: kube-discovery
+    <master/addons> created essential addon: kube-proxy
+    <master/addons> created essential addon: kube-dns
+
+    Kubernetes master initialised successfully!
+
+    You can connect any number of nodes by running:
+
+    kubeadm join --token f0c861.753c505740ecde4c 138.68.135.192
+
+Make a record of the `kubeadm join` command that `kubeadm init` outputs.
+You will need this in a moment.
+The key included here is secret, keep it safe &mdash; anyone with this key will be able to add authenticated nodes to your cluster.
+
+The key is used for mutual authentication between the master and the joining nodes.
+
+### Joining your nodes
+
+The nodes are where your containers (your workload) will run.
+To add a new machine as a node to your cluster, SSH to that machine, become root (e.g. `sudo su -`) and run the command that was output by `kubeadm init`.
+For example:
+
+    # kubeadm join --token f0c861.753c505740ecde4c 138.68.135.192
+
+A few seconds later, you should notice that running `kubectl get nodes` on the master shows a cluster with as many machines as you created.
+Your cluster is now bootstrapped!
+
+But before you can deploy applications to it, you need to install a pod network.
+
+### Installing a pod network
+
+You must install a pod network add-on so that your pods can communicate with eachother on different hosts.
+
+Several projects provide Kubernetes pod networks.
+A simple one with no infrastructure or database dependencies is Weave Net, which you can install with:
+
+    # kubectl apply -f https://git.io/weave-kube
+
+You can learn more about Weave for Kubernetes on the project's [GitHub page](https://github.com/weaveworks/weave-kube).
+
+You can see a complete list of available network add-ons on the [add-ons page](/docs/admin/addons/).
```

Comment:
+1 I love all of the work y'all are doing. But please lets centralize the docs to a single list and then link off to the external docs for installation. These docs are going to sprawl and break if we don't focus it to a single location.